public/index.css

```diff
@@ -165,41 +184,32 @@ h5 {
 		flex-direction: row-reverse;
 		justify-content: flex-end;
 		align-items: baseline;
-		margin-inline-start: calc(-1 * var(--icon-size));
+		margin-inline-start: calc(-1 * var(--icon-size-x));
 	}
 
 	.heading-wrapper > .anchor-link {
-		inset-inline-start: -0.75rem;
-	}
-
-	.heading-wrapper .anchor-icon {
-		position: relative;
-		bottom: var(--icon-baseline, 0);
-	}
-
-	.heading-wrapper.level-h2 .anchor-icon {
-		--icon-baseline: 0.35rem;
+		margin-inline-end: 0;
 	}
+}
 
-	.heading-wrapper.level-h3 .anchor-icon {
-		--icon-baseline: 0.15rem;
-	}
+.heading-wrapper > .anchor-link {
+	opacity: 0;
 }
 
-@media (hover: hover) {
+@media (hover: none) {
```

Comment:
Sorry I thought you'd done but noticed we're relying on this media query now instead of `hover: hover`. On some phones (Sarah's for example) the `hover` media query isn't set so the hover links are never revealed. We need to use the inverse logic so:

1. reveal by default
2. progressively enhance by hiding links for devices that say they are hover-capable

Reply:
Oh wow, I didn't know that there are devices that don't follow this part of the spec (yet?), but still understand hover: hover. I use this media query in multiple locations and will recheck, thanks!

Reply:
Done. As there were too many locations and too much logic to rewrite otherwise, I just used the actual inverse query, which also does what you described.

Reply:
Will that still work if the hover media flag isn't provided by the user agent? I'll trust you but we should wait for @sarah11918 to confirm on her devices as we know it was an issue on those. Thanks!

Reply:
Ohhh maybe I misunderstood your first post above. I don't think Sarah's phone _would_ understand `hover: hover` so that's why we can use it. A media query that isn't known/understood is just ignored so: reveal by default, hide only for browsers that understand and match the query.

Reply:
The `not` queries are special - if their contents do not match or are not understood by the browser, their contents are applied. So we should be perfectly fine. :)

Reply:
Ok! 🤞

Reply:
Ok, so I just tested it in Chrome, and it appears that I was wrong. When I insert a rule like "not screen and (yabba: hut)", it should always apply occording to my logic. However, it doesn't apply. So I'll actually have to rework all the rules. Stay tuned! 😁

Reply:
🙌

Reply:
It's properly fixed now. :)